org.mixedrealitytoolkit.input/Interactors/GazePinch/GazePinchInteractor.cs

```diff
@@ -444,5 +452,38 @@ protected override void OnSelectExited(SelectExitEventArgs args)
         }
 
         #endregion XRBaseInteractor
+
+        /// <summary>
+        /// Updates the pinch state of the GazePinchInteractor.
+        /// If handedness is not set then it defaults to right hand.
+        /// If the pinch data is not available for the set hand then the other hand is tried.
+        /// </summary>
+        private void UpdatePinchState()
+        {
+            if (logicalSelectState == null)
+            {
+                Debug.LogWarning("GazePinchInteractor is missing logicalSelectState, pinch state won't update.");
+                return;
+            }
+
+            if (XRSubsystemHelpers.HandsAggregator == null)
+            {
+                Debug.LogWarning("XRSubsystemHelpers.HandsAggregator is null, pinch state won't update.");
+                return;
+            }
+
+            bool gotPinchData = XRSubsystemHelpers.HandsAggregator.TryGetPinchProgress(handedness.ToXRNode(),
+                out bool isPinchReady, out bool isPinching, out float pinchAmount);
+            if (!gotPinchData) //Try the other hand if the set hand does not have pinch data.
+            {
+                gotPinchData = XRSubsystemHelpers.HandsAggregator.TryGetPinchProgress(handedness.ToXRNode() == XRNode.LeftHand ? XRNode.RightHand : XRNode.LeftHand,
```

Comment:
you're converting the value twice.  store in `handedness.ToXRNode()` in local var.

Reply:
Right, fixed in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/743/commits/386494ad8a6d2591fb95d72335487b106ee6e339 , resolving.